README.md

```diff
@@ -8,26 +8,13 @@ My personal portfolio site
 
 - HTML
 - CSS
+- JAVASCRIPT
 
 
 ## Getting Started
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
-
 ## Authors
 
 👤 **Cindy Dorantes**
```

Comment:
- [ ] Kindly make sure you add the `MIT.md` licensed file as it makes the MIT Licensed link at the bottom broken. I suggest you take it from [this template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) and add it to your root directory. 
- [ ] Also, make sure you improve your README documentation to make it professional by completing the important sections of your README like **adding information about cloning the repo locally**, **Update your project description in the way it describes exactly the project, etc**. You can use [this README as a template](https://github.com/tresorsawasawa/MyPortfolio/blob/develop/README.md) to help you with how to fill some sections.